src/test/shell/bazel/skylark_git_repository_test.sh

```diff
@@ -158,6 +161,27 @@ EOF
   expect_log "Pluto is a dwarf planet"
 }
 
+# This test:
+# 1. Creates a git_repository rule with some commit hash
+# 2. Make sure worksapce can be built and repository cache is updated
+function test_git_repositry_cache_is_populated() {
+  local pluto_repo_dir=$TEST_TMPDIR/repos/pluto
+  local cache_dir=/tmp/bazel_cache
+  local commit_hash="52f9a3f87a2dd17ae0e5847bbae9734f09354afd"
+
+
+  export BAZEL_GIT_REPOSITORY_CACHE=$cache_dir
+
+  # Do not run tests if the git worktree command does not exist.
+  # Testing without if check - is windows timing out due to that?
+  # if git worktree --help; then
```

Comment:
Why is this condition commented out? Should it be there? Or should we actually run the test even in this case, and just skip the verification that the correct hash ended up in the cache.